configman/converters.py

```diff
@@ -42,12 +42,11 @@
 import types
 import json
 
-from required_config import RequiredConfig
-from namespace import Namespace
-
-from .datetime_util import datetime_from_ISO_string as datetime_converter
-from .datetime_util import date_from_ISO_string as date_converter
-from .config_exceptions import CannotConvertError
+from configman.datetime_util import (
+    datetime_from_ISO_string as datetime_converter
+)
+from configman.datetime_util import date_from_ISO_string as date_converter
```

Comment:
If you like that name better why not change it in `datetime_util.py`?

And if you worry about backwards compatibility add another alias function in `datetime_util.py`